source/Common - Copy/Common.csproj

```diff
@@ -0,0 +1,119 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This class (and everything in this directory) was in a branch that Joke rejected: https://github.com/Bannerlord-Coop-Team/BannerlordCoop/pull/438
You'll essentially want to revert this commit on your branch: https://github.com/Bannerlord-Coop-Team/BannerlordCoop/pull/438/commits/0ab2608ef3761917187fab55cf193e15a77f2be1